tool/tsh/help.go

```diff
@@ -21,6 +21,9 @@ const (
 	loginUsageFooter = `NOTES:
   The proxy address format is host:https_port,ssh_proxy_port
 
+  Passwordless only works in local auth
+  --auth=passwordless flag can be omitted if cluster if set to default to passwordless
```

Comment:
I think it helps to be more specific here:

```suggestion
  --auth=passwordless flag can be omitted if your cluster configuration set the connector_name: passwordless option.
```